### PR TITLE
Configurable wait threshold for impaired nodes

### PR DESCRIPTION
**Pull request by FinchPowers**

### CODE REVIEW

File: starcluster/cluster.py [hunk 1]
```diff
@@ -2067,16 +2070,52 @@ def get_impaired_nodes(self):
...
+            }
+
+        if not self._impaired_nodes:
+            # nothing to do
+            return
+
+        log.info("Currently impaired nodes: "
+                 + str(self._impaired_nodes.keys()))
+        log.info("Previously impaired nodes: "
+                 + str(previously_impaired_nodes.keys()))
```
Comment (FinchPowers) [inline starcluster/cluster.py]: Don't show that now, unfiltered list. Filter it and then display it with the time.

**Comment by jraby** [comment]
If I read this correctly, the configurable threshold is set to 120 seconds by default?
Is that long enough or are we expected to bump that to some high value (5 minutes or something).

In anycase, I'd say we're better off setting the default value to a known good value and avoid messing with it (I doubt we'll be actively tweaking this once we find the sweet spot).

**Comment by FinchPowers** [comment]
Correct. From my observations over the last week in production it never reached 2 minutes. I had to set it to 10 seconds to test the case where it does something.

**Comment by jraby** [comment]
Alright, then let's set it to a reasonable value and merge this!

**Comment by FinchPowers** [comment]
Squash merged.

**Pull request closed without merging**